src/Parser.js

```diff
@@ -518,64 +504,50 @@ export default class Parser {
         type: ?ArgType,
         optional: boolean,
         greediness: ?number,
-        consumeSpaces: boolean,
     ): ?AnyParseNode {
         switch (type) {
             case "color":
-                if (consumeSpaces) {
-                    this.consumeSpaces();
-                }
```

Comment:
Where did all of these calls to `consumeSpaces()` go?

Reply:
They have been moved to match TeX behavior: https://github.com/KaTeX/KaTeX/blob/35470fd7db11202e01d3097bbc57fe2a1eb67acd/src/MacroExpander.js#L115-L120 https://github.com/KaTeX/KaTeX/blob/35470fd7db11202e01d3097bbc57fe2a1eb67acd/src/MacroExpander.js#L155-L170